sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -28,16 +28,20 @@ public partial class KeyVaultAccessControlClient
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultRoleDefinition>> DeleteRoleDefinitionAsync(System.Guid roleDefinitionName, Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultRoleAssignment> GetRoleAssignment(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, string roleAssignmentName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultRoleAssignment>> GetRoleAssignmentAsync(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, string roleAssignmentName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.Security.KeyVault.Administration.KeyVaultRoleAssignment> GetRoleAssignments(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.Security.KeyVault.Administration.KeyVaultRoleAssignment> GetRoleAssignmentsAsync(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.Security.KeyVault.Administration.KeyVaultRoleDefinition> GetRoleDefinitions(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.Security.KeyVault.Administration.KeyVaultRoleDefinition> GetRoleDefinitionsAsync(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Security.KeyVault.Administration.KeyVaultRoleDefinition> UpsertRoleDefinition(string roleDefinitionDescription, Azure.Security.KeyVault.Administration.KeyVaultPermission permissions, Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope roleScope = default(Azure.Security.KeyVault.Administration.Models.KeyVaultRoleScope), System.Guid? roleDefinitionName = default(System.Guid?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Is "Upsert" okay to use? In search, we use "CreateOrUpdate" as a prefix. @tg-msft @schaabs @KrzysztofCwalina what do you think? I don't see either in https://azure.github.io/azure-sdk/dotnet_introduction.html#sync-and-async.

Reply:
@maorleger asked about this at https://github.com/Azure/azure-sdk-for-js/issues/12206#issuecomment-769482971 and I got to thinking there: `CreateOrUpdate` is quite obvious in its intent, where some users may not know what `Upsert` means. I still defer to @KrzysztofCwalina (@tg-msft did offline as well) but we should get it recorded in our guidelines. Seems [TypeScript allows](https://azure.github.io/azure-sdk/typescript_design.html#ts-client-naming-conventions) `upsert` as well as `set`, which is another option to consider here. I think that one is less obvious than `CreateOrUpdate`, though.